types/ember-data__store/ember-data__store-tests.ts

```diff
@@ -30,3 +30,4 @@ Store; // $ExpectType<DS.Store>
 Snapshot; // $ExpectType<DS.Snapshot>
 normalizeModelName("post"); // $ExpectType<string>
 normalizeModelName("post-comment"); // $ExpectType<string>
+// recordIdentifierFor(Store.createRecord("post", { title: "title" })); // $ExpectType<RecordIdentifier<"post">>
```

Comment:
This test doesn't work because I'm not able to actually create a record. Is there a workaround? If not, I'll just remove this commented out test.